sherpa/models/regrid.py

```diff
@@ -649,6 +649,11 @@ def _make_and_validate_grid(self, args_array):
             raise ModelErr('needsint')
         if requested_eval_space.is_integrated and not self.evaluation_space.is_integrated:
             raise ModelErr('needspoint')
+        if self.evaluation_space.is_integrated and requested_eval_space.is_integrated:
+            lo = self.evaluation_space.grid[0]
+            hi = self.evaluation_space.grid[1]
+            if np.any(lo[1:] < hi[:-1]):
```

Comment:
Do we need to worry about numerical effects here? I am pretty sure that with some of the "does this ARF make sense" checks we could be given grids that slightly overlapped, so there was some effort to support this.

I actually don't think we want to here (i.e. that the existing code is sufficient), but wanted to discuss it. My thought here is that the grid here is almost-certainly user defined (i.e. not from an external definition such as the ARF or RMF) so there's 

 - less likelihood of getting overlapping bins
 - it is much easier for a user to correct the grid here so that it doesn't overlap compared to the ARF/RMF case

Reply:
Agree.